test/libraries/Position.t.sol

```diff
@@ -21,4 +22,41 @@ contract PositionTest is Test {
         }
         assertEq(positionSlot, expectedPositionSlot, "slots not equal");
     }
+
+    function test_fuzz_update(
+        int128 liquidityDelta,
+        uint256 feeGrowthInside0X128,
+        uint256 feeGrowthInside1X128,
+        uint128 liquidity
+    ) public {
+        // grab a position and set its values
+        Position.Info storage position = positions[0];
+        position.liquidity = liquidity;
+        position.feeGrowthInside0LastX128 = feeGrowthInside0X128;
+        position.feeGrowthInside1LastX128 = feeGrowthInside1X128;
+        if (position.liquidity == 0 && liquidityDelta == 0) {
+            vm.expectRevert(Position.CannotUpdateEmptyPosition.selector);
+        }
+        uint128 oldLiquidity = position.liquidity;
+        uint256 oldFeeGrowthInside0X128 = position.feeGrowthInside0LastX128;
+        uint256 oldFeeGrowthInside1X128 = position.feeGrowthInside1LastX128;
+
+        // new liquidity cannot overflow uint128
+        uint256 newLiquidity = uint256(int256(uint256(oldLiquidity)) + int256(liquidityDelta));
+        if (newLiquidity > type(uint128).max) {
+            vm.expectRevert(SafeCast.SafeCastOverflow.selector);
+        }
+
+        Position.update(position, liquidityDelta, feeGrowthInside0X128, feeGrowthInside1X128);
```

Comment:
feels like theres quite a few things not being tested here
- the feeGrowth being passed in is the same as the one already in the struct. That means its not testing whether the fee growth is getting updated
- `update` returns 2 values that are calculated by the function that are not being checked